conconartist/faceit-sql-server
# Setup Express server

## What is the change?

Sets up initial database server with Express for the faceit app (https://github.com/conconartist/face-it), using mock data and a simplified table.  The server can be accessed locally when downloaded to the user's machine.  A `GET` request can be made successfully in Postman.  

 ## Is this a fix or a feature? 

Feature

 ## Where should the reviewer start? 

Server.js and productsData.js to view data.  

## How should this be tested?

1. Download this repo to your local machine by forking or clicking on the green code button on the front page of the repo to copy the SSH key.  In your terminal, type `git clone` before pasting the SSH key and press `enter`.
2. Download the necessary dependencies by typing `npm i` in the next command line of the terminal and press `enter`.
3. When your download is complete, open the server in your script editor to view code.  
4. To start the server, enter `node server.js` in the terminal and press enter.  You should see a message that says "Now listening on port 3002!"
5. Go to `http://localhost:3002` in your browser URL to view the server.
6. If you have Postman, create a `GET` request for `http://localhost:3000/` to view the data in the response.
